### ISSUE TITLE
grafana-toolkit plugin:build and plugin:dev failed in windows

### ISSUE THREAD

**Issue by peterhpchen** [grafana/grafana#18294]
Because the difference of path system between Linux and Windows. The code below would get error in Windows OS:

https://github.com/grafana/grafana/blob/7949329636a132b811bcc2c1009ee32e40ddbefb/packages/grafana-toolkit/src/config/webpack.plugin.config.ts#L45

The result of `path.resolve`:
* In Linux, the path separator is `/`.
* In Windows, it is `\`.

So `/src/` is never found in Windows.

**What happened**:
`grafana-toolkit plugin:build` and `plugin:dev` failed.

**What you expected to happen**:
`grafana-toolkit plugin:build` and `plugin:dev` success.

**How to reproduce it (as minimally and precisely as possible)**:
Execute `grafana-toolkit plugin:dev` or `plugin:build` on any plugin in Windows.

**Anything else we need to know?**:
It can be resolved by use regex.

```js
var match = id.match(/([\\\/])src\1/i);
var idx = id.indexOf(match[0]);
```

**Environment**:
- @grafana/toolkit version: **6.4.0-alpha.41**
- User OS & Browser: **Windows 10**

**Comment by ryantxu** [comment]
Fixed in #18306

**Comment by ryantxu** [comment]
@peterhpchen -- can you try again with 6.4.0-alpha.42?

**Comment by peterhpchen** [comment]
This bug is fixed on 6.4.0-alpha.42 in my environment.

Thank you very much. @ryantxu
